docs: Troubleshooting cifuzz headers not found (#633)
CodeIntelligenceTesting/cifuzz: docs/Getting-Started.md

Also, watch [Going Beyond Unit Testing | How to Uncover Blind Spots in your
Java Code with Fuzzing](https://www.youtube.com/watch?v=8yECb-p3cQI) on
YouTube.

## Troubleshooting

> If problems occur when installing or running cifuzz, you can check the [Troubleshooting](Troubleshooting.md)
> for possible solutions.

Reviewer: ```suggestion
If you encounter problems installing or running cifuzz, you can check [Troubleshooting](Troubleshooting.md)
for possible solutions.
```
